tests/integration/security/cacert_rotation/main_test.go

```diff
@@ -109,30 +109,32 @@ func TestReachability(t *testing.T) {
 					from.CallOrFail(t, opts)
 				})
```

Comment:
you'd better verify traffic after cert rotated.

Reply:
weird, i must be mistaken

Reply:
I checked have also verified at last in https://github.com/istio/istio/pull/47959/files#diff-62a162ca6a83223a77d1f5af4a8499e940253095d5a2b53bc174ab53ced120e6R140-R156